frontend/packages/console-shared/src/promql/cluster-dashboard.ts

```diff
@@ -309,7 +309,10 @@ const overviewQueries = {
   [OverviewQuery.CPU_UTILIZATION]: _.template(
     `
       sum(
-        1 - sum without (mode) (rate(node_cpu_seconds_total{mode=~"idle|iowait|steal"}[2m]))
+          (
+            sum without (mode) (rate(node_cpu_seconds_total{}[2m]))
```

Comment:
if we want to go that way, we could just `node_cpu_seconds_total{mode!~"idle|iowait|steal"}`?

I think the reason we use a hardcoded `1` is that due to sampling, other approximations, or because some states aren't covered, the overall sum isn't expected to be 1.

Using the sum would help hide/balance things out, but maybe the problem is that numbers in `/proc/[pid]/stat` are wrong or node-exporter is misinterpreting them in some occasions...

Reply:
Yeah you're right of course...not sure why I didn't see that.

> I think the reason we use a hardcoded 1 is that due to sampling, other approximations, or because some states aren't covered, the overall sum isn't expected to be 1.

Yeah I agree. The issue here though is that sometimes even `sum without (mode) (rate(node_cpu_seconds_total{mode=~"idle|iowait|steal"}[2m]) > 1` so the overall sum becomes negative and thus the graph in the console dips into the negative. So hardcoding the 1 doesn't really help in normalizing this to 1 does it?

Reply:
yes, using sum() (instead of 1) would make those large values on `mode=~"idle|iowait|steal"` cancel each other out. But I think we need to understand why we get those large values; enable debug level on node-exporter, continually dump `/proc/[pid]/stat` outside of node-exporter to see if `/proc` is faulty or if node-exporter does bad conversions.